src/pages/meta-chain/index.page.tsx

```diff
@@ -37,20 +39,27 @@ export default function MetaChainPage() {
         />
       </div>
       <MetaChainKeyFeatures />
+      <MetaChainRoadmap roadMapImg={roadMapImg} />
     </>
   );
 }
+async function getRoadMapFromPrismic(): Promise<any> {
```

Comment:
dont see any `await` in the function, do we need `async`?

Reply:
yes, `endpoint.getByType("metachain");` requires an `await`